src/app/pages/ArticlePage/PagePromoSections/TopStoriesSection/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useContext } from 'react';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { arrayOf, shape } from 'prop-types';
+import { storyItem } from '#models/propTypes/storyItem';
+import pathOr from 'ramda/src/pathOr';
+import { C_GREY_2 } from '#app/legacy/psammead-styles/src/colours';
+import SectionLabel from '#legacy/psammead-section-label/src';
+import {
+  StyledWrapper,
+  FlexPromoList,
+  FlexPromoListItem,
+} from './index.styles';
+import TopStoriesItem from './TopStoriesItem';
+
+const TopStoriesPromo = ({ content }) => {
```

Comment:
```suggestion
const TopStoriesSection = ({ content }) => {
```
To be consistent with file rename

Reply:
Done